### PR TITLE
#183031478 - USER SHOULD BE ABLE TO REGISTER

### PR DESCRIPTION
**Pull request by dalienst**
#### What does this Pull Request do?

- Enables user to sign up
- Connects backend register endpoint to the frontend

#### Description of Task to be completed?

- sign up for an account
- validate user inputs

#### How should this be manually tested?

- run `npm start`
- go to the sign up page
- fill in the fields
- submit

#### Any background context you want to provide?

- The user should be able to create a new account. The form should validate the user's input and raise an error when the inputs are invalid. A success message is returned when registration is successful

#### What are the relevant pivotal tracker story?

- [183031478](https://www.pivotaltracker.com/n/projects/2570604/stories/183031478)

#### Screenshots (if appropriate)

image

Screenshot from 2022-08-25 14-25-03

#### Questions:

- N/A

### CODE REVIEW

File: src/assets/styles/main.scss [hunk 1]
```diff
@@ -6,3 +6,6 @@
 @import 'features';
 @import 'base';
 @import 'abstracts';
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (KaigaMunene) [inline src/assets/styles/main.scss]: newline after this

File: src/assets/styles/pages/_index.scss [hunk 2]
```diff
@@ -1,2 +1,2 @@
 @import 'home';
-@import 'login';
+@import 'login';
```
Comment (KaigaMunene) [inline src/assets/styles/pages/_index.scss]: newline after this

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (KaigaMunene) [inline src/index.css]: newline after this

File: src/assets/styles/main.scss (hunk 1 above)
Comment (dalienst) [reply src/assets/styles/main.scss]: Done

File: src/pages/Registration.jsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+import { toast } from 'react-toastify';
+import { RegistrationSchema } from '../validation/Registration';
+import axios from '../api/axios';
+import { urls, publicLinks } from '../constants/links';
+
+function Registration() {
+  const navigate = useNavigate();
+  return (
+    <div className="w-full flex justify-center p-10 bg-gray-300 h-screen">
+      <Formik
```
Comment (KaigaMunene) [inline src/pages/Registration.jsx:14]: write descriptive names "formik ni"

Comment (dalienst) [reply src/pages/Registration.jsx:14]: That is how it should be. It is a package to help in form validation.

**Pull request merged**